js/plugins/jsqueue-draggable.js

```diff
@@ -25,7 +25,23 @@ function jsqueue_draggable() {
 
     this.MAKE_SORTABLE = function (data) {
         var self = this;
+
+        if (data['file_upload_event']) {
```

Comment:
Make this more generic. So recieve_event. So basically we can reuse this else where and it makes sense